
Training with beta_scale: 1e-05 and penalty_scale: 1e-10


Step 30 of 500; Loss: 1.2353e+05. Model Size: 7714 (Time: 41.7s)
Exception ignored in: <function _xla_gc_callback at 0x7955d99ddee0>
Traceback (most recent call last):
  File "/home/rsw0/miniforge3/envs/yolanda/lib/python3.9/site-packages/jax/_src/lib/__init__.py", line 96, in _xla_gc_callback
    def _xla_gc_callback(*args):
KeyboardInterrupt:
Exception ignored in: <function _xla_gc_callback at 0x7955d99ddee0>
Traceback (most recent call last):
  File "/home/rsw0/miniforge3/envs/yolanda/lib/python3.9/site-packages/jax/_src/lib/__init__.py", line 96, in _xla_gc_callback
    def _xla_gc_callback(*args):
KeyboardInterrupt:
Exception ignored in: <function _xla_gc_callback at 0x7955d99ddee0>
Traceback (most recent call last):
  File "/home/rsw0/miniforge3/envs/yolanda/lib/python3.9/site-packages/jax/_src/lib/__init__.py", line 96, in _xla_gc_callback
    def _xla_gc_callback(*args):
KeyboardInterrupt:
Exception ignored in: <function _xla_gc_callback at 0x7955d99ddee0>
Traceback (most recent call last):
  File "/home/rsw0/miniforge3/envs/yolanda/lib/python3.9/site-packages/jax/_src/lib/__init__.py", line 96, in _xla_gc_callback
    def _xla_gc_callback(*args):

